db-client/src/main/java/com/splicemachine/db/client/am/SQLExceptionFactory40.java

```diff
@@ -64,7 +57,7 @@
      * @param errCode db error code
      */
     public SQLException getSQLException (String message, String sqlState, 
-                                                            int errCode) { 
+                                                            int errCode) {
```

Comment:
While you're at it, could you make it consistent with the newly-added method below and indent `int errCode`?